### PR TITLE
Update dog pregnancy constraints

### PR DESCRIPTION
**Pull request by btjl**

### CODE REVIEW

File: tests/admin/get-incomplete-profiles.test.ts [hunk 1]
```diff
@@ -52,7 +52,7 @@ describe("getIncompleteProfiles", () => {
       });
       const profile = guaranteed(result)[0];
       const expected = await getExpectedProfile(1, userId, dogId);
-      expect(profile).toEqual(expected);
+      expect(profile).toEqual(expected); //
```
[the remark this answers is not in the document]
Comment (btjl) [reply tests/admin/get-incomplete-profiles.test.ts]: Done fbbc9aa74cf15c2f969440b2e09249e4521c5279

File: tests/schema/dog-statuses.test.ts [hunk 2]
```diff
@@ -156,6 +160,7 @@ describe("dog_statuses view", () => {
           dogSpec: {
             dogBreed: "Test Dog",
             dogWeightKg: 18,
+            dogGender: DOG_GENDER.UNKNOWN,
```
[the remark this answers is not in the document]
Comment (btjl) [reply tests/schema/dog-statuses.test.ts]: Done fbbc9aa74cf15c2f969440b2e09249e4521c5279

File: tests/schema/dog-statuses.test.ts [hunk 3]
```diff
@@ -285,6 +294,7 @@ describe("dog_statuses view", () => {
         const { dogId } = await initDog(dbPool, {
           dogSpec: {
             ...ELIGIBLE_SPEC,
+            dogGender: DOG_GENDER.UNKNOWN,
```
[the remark this answers is not in the document]
Comment (btjl) [reply tests/schema/dog-statuses.test.ts]: Done fbbc9aa74cf15c2f969440b2e09249e4521c5279

**Review by danui — APPROVED** [review]
LGTM

**Pull request merged**